theme/ckeditor5-ui/globals/_fonts.css

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-font-size-base: 12px;
+	--ck-line-height-base: 1.67;
+	--ck-font-face: Helvetica, Arial, Tahoma, Verdana, Sans-Serif;
+
+	--ck-font-size-tiny: 0.7em;
+	--ck-font-size-small: 0.75em;
+	--ck-font-size-normal: 1em;
+	--ck-font-size-big: 1.4em;
+	--ck-font-size-large: 1.8em;
+}
```

Comment:
We are setting levels of property with `calc` in `theme/ckeditor5-ui/globals/_spacing.css` maybe we should use it here?

Reply:
Probably a good idea.

Reply:
OTTH I noticed that while spacings multiply `.8em`, which is not super-easy, fonts are related to `1em`, which is pretty straightforward.  Changing it to `calc` would be

```css
--ck-font-size-big: 1.4em;
```
vs
```css
--ck-font-size-big: calc(var(--some-base-size) * 1.4));
```

which looks bloated. I don't think it's a good idea.